vehicles/src/modules/credit-account/credit-account-user.controller.ts

```diff
@@ -65,6 +66,10 @@ export class CreditAccountUserController {
     description: 'The result of the changes to the credit account user.',
     type: ReadCreditAccountUserDto,
   })
+  @ApiUnprocessableEntityResponse({
+    description: 'Cannot process the Add/Update credit account user request.',
+    type: ExceptionDto,
+  })
   @Put()
   @Roles(Role.WRITE_CREDIT_ACCOUNT)
   async addOrActivateCreditAccountUser(
```

Comment:
What does "activate" mean here?

Reply:
If there is an existing relation between Credit Account User and Credit Account in inactive status, then update it as Active.